doc/nrf-bm/includes/softdevice_flash.txt

```diff
@@ -8,19 +8,19 @@ To flash the S115 SoftDevice binary to your device, open the terminal and run th
 
          .. code-block:: console
 
-            cd C:/ncs/bm_v0.7.0/nrf-bm
+            cd C:/ncs/nrf-bm/v0.8.0
```

Comment:
For later: can we set the version elsewhere and reuse? To avoid the need of updating it many places.

Reply:
Not in code blocks AFAIK